app/points/calculate-weighting.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (weightingPercentage) {
+  var weighting
+  if (weightingPercentage >= 100) {
+    weighting = 0
+  } else if (weightingPercentage <= 0) {
+    weighting = 1
+  } else {
+    weighting = (100 - weightingPercentage) / 100
+  }
+  return weighting
+}
```

Comment:
We could, I think single exit point is slightly easier to read though. I'm happy to go with the majority on this though, @scoch what's your preference?